.eslintrc

```diff
@@ -59,10 +62,8 @@
     "react/function-component-definition": [2, {
       "namedComponents": "arrow-function"
     }],
-    "react/hook-use-state": "error",
     "react/no-array-index-key": "error",
     "react/no-multi-comp": "error",
-    "react/require-default-props": "error",
```

Comment:
I think we should keep these rules:
`hook-use-state` makes sure `useState` values are named consistently
and `require-default-props` makes sure that all not required props have a default value. This is especially important when a function prop is not required, so to avoid calling `undefined` function we should pass `noop` as a default value

Reply:
If we use the `require-default-props` rule and pass `noop` as the default value, the `renderCustomTrigger` prop will always be `true`.
```
const renderTrigger = () => {
    return renderCustomTrigger
      ? renderCustomTrigger({ onClick: openModal })
      : renderDefaultTrigger();
  };
```
The `hook-use-state` rule shows errors in `index.js` files and not only.

Reply:
@Dmytro-Melnyshyn, ok, and in this case we can pass `null` as default value - this way we can both keep the eslint rule, and `null` value will be handled inside component with ternary operator

What error does `hook-use-state` show?

Reply:
![image](https://user-images.githubusercontent.com/77053927/181259956-7501324f-5b61-4cf9-b03d-889cd5987dab.png)

Reply:
@Dmytro-Melnyshyn oh it seems we're missing `eslint-plugin-react` in `devDeps`

Reply:
You are right, thanks!